src/universe.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+// This is supplemental/static list of repos. These don't appear in the extension-feed.
+
+return function() {
+  $result = array(
+    'civicrm-core' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-core',
+      'type' => 'core',
+    ),
+    'civicrm-backdrop-1.x' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-backdrop',
+      'git_branch' => '1.x-master',
+      'type' => 'core',
+    ),
+    'civicrm-buildkit' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-buildkit',
+      'type' => 'tools',
+    ),
+    'civicrm-drupal-6.x' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-drupal',
+      'git_branch' => '6.x-master',
+      'type' => 'core',
+    ),
+    'civicrm-drupal-7.x' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-drupal',
+      'git_branch' => '7.x-master',
+      'type' => 'core',
+    ),
+    'civicrm-drupal-8.x' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-drupal-8',
+      'type' => 'core',
+    ),
+    'civicrm-joomla' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-joomla',
+      'type' => 'core',
+    ),
+    'civicrm-packages' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-packages',
+      'type' => 'core',
+    ),
+    'civicrm-setup' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-setup',
+      'type' => 'lib',
+    ),
+    'civicrm-wordpress' => array(
+      'git_url' => 'https://github.com/civicrm/civicrm-wordpress',
+      'type' => 'core',
+    ),
+    'civihr' => array(
+      'git_url' => 'https://github.com/civicrm/civihr',
+      'git_branch' => 'staging',
+      'type' => 'ext',
+    ),
+    'com.webaccessglobal.module.civimobile' => array(
+      'git_url' => 'https://github.com/webaccess/com.webaccessglobal.module.civimobile',
+      'type' => 'ext',
+    ),
+    'civix' => array(
+      'git_url' => 'https://github.com/totten/civix',
+      'type' => 'tools',
+    ),
+    'coworker' => array(
+      'git_url' => 'https://lab.civicrm.org/dev/coworker',
+      'type' => 'tools',
+    ),
+    'cf-civicrm' => array(
+      'git_url' => 'https://github.com/mecachisenros/cf-civicrm',
```

Comment:
@totten This repo is abandoned.  This repo - https://github.com/WPCV/cf-civicrm  is the fork with patches to work with CiviCRM 5.41+

Reply:
I missed that this was merged.   I'll try and send up a PR when I get a moment

Reply:
@kcristiano Yup, that would be good. Maybe those other repos from `WPCV` should be added as well?